Gemfile

```diff
@@ -71,6 +72,7 @@ group :development do
   gem 'capistrano-rails', require: false
   gem 'capistrano-rvm', require: false
   gem 'capistrano3-puma', require: false
+  gem 'capistrano-yarn'
```

Comment:
INFO: Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `capistrano-yarn` should appear before `capistrano3-puma`.